frontend/src/sass/pages/_application-view.scss

```diff
@@ -135,23 +149,39 @@
   display: flex;
   flex-direction: row;
   align-items: flex-end;
+  @media (max-width: 600px) {
+    position: unset;
+    display: flex;
+    flex-direction: column;
+    align-items: unset;
+  }
+}
 
+.phone-extension-wrapper {
+  position: absolute;
+  top: 2.2rem;
+  left: 30rem;
   @media (max-width: 600px) {
-     position: unset;
-     display: flex;
-     flex-direction: column;
-     align-items: unset;
-   }
+    position: unset;
+    top: unset;
+    left: unset;
+  }
 }
 
-  .phone-extension-wrapper {
-    position: absolute;
-    top: 2.2rem;
-    left: 30rem;
+.required-documents {
+  background: white;
+  border: 1px solid black;
+  padding: 0 35px 35px;
```

Comment:
Do we wanna use rem here since most of the code-base is using rem over px?

Reply:
We are actually using a variety of unit types throughout the application, including `px` in several other locations.

But this is a good intention for sure, and points at a potentially larger issue: we should consider implementing a relative unit such as `rem` throughout the entire codebase. This would make the appearance more visually consistent across all devices, as well as keep our codebase consistent in regard to this unit type.

I think we should consider bringing this up with other folks on the team and assess if it's worth creating an issue to implement a consistent unit type across the entire codebase.